### DISCUSSION TITLE
Should npm run debug result in bundle-js error?

### DISCUSSION THREAD

**Discussion by szachy113** [darkreader/darkreader#10663] [General]
When running the `npm run debug` command, I see the following error:

<img width="837" alt="Screenshot 2023-01-20 at 16 29 54" src="https://user-images.githubusercontent.com/81178959/213736901-959cbbf2-1acc-41ee-9f0d-11990acf6aa3.png">

Should it be the case?

Thanks in advance! 🙏🏻

**Comment by Gusted** [comment]
No that shouldn't happen. I'm not sure where the cache file is stored, but the error indicates it has no permission to write to the cache directory of rollup.
